### DISCUSSION TITLE
Bugs on WeedWallet

### DISCUSSION THREAD

**Discussion by kjincgroup** [BitCannaGlobal/WeedWallet#58] [General]
Found below bugs on WeedWallet

1.  Account name on Portfolio page(Simple Mode) is incorrect, should be showing the account name in Keplr?
![image](https://github.com/BitCannaGlobal/WeedWallet/assets/111372349/adf4dac5-21a5-4932-9779-ae5feeeb4676)

2. Cannot display unjail transaction in Transactions page
![image](https://github.com/BitCannaGlobal/WeedWallet/assets/111372349/1782012c-6108-40f6-8907-7f2f66a8a713)

**Comment by kjincgroup** [comment]
Receive, Stake and Claim button are not working(No response) in Portfolio Page(Simple Mode)

![image](https://github.com/BitCannaGlobal/WeedWallet/assets/111372349/f4c75f89-e609-418c-a953-2f40b3ec3145)
![image](https://github.com/BitCannaGlobal/WeedWallet/assets/111372349/7c126301-cc48-4094-8824-ada4aa223c91)

**Comment by TJ-BitCanna** [comment]
Thanks! We'll take these with us and convert to issues early next week.

Stay tuned for this one to help testing and see if it is ok.

**Comment by TJ-BitCanna** [comment]
Suggested issue:
- on the Simple page, make sure the account name is retrieved from Keplr
- debug "Unjail" transactions to be shown in the Transactions page (also check "Create Validator" and "Edit Validator")
- on the Simple page, make sure the "Claim", "Receive", "Stake" buttons are working

#102

**Comment by TJ-BitCanna** [comment]
Issues are solved :)
